SubmissionTest.java

```diff
@@ -909,6 +909,77 @@ public void testGetSoundBlocksForProgramEmpty()
     }
 
     /**
+     * Test PopulateGlobalVariables - valid.
+     */
+    @Test
+    public void testPopulateGlobalVariables()
+    {
+        String[] expected = {"name", "weather", "sprite1AllVariables"};
+        String[] actual = submissions[1].getGlobalVariables();
+        assertEquals("should be equal", expected, actual);
```

Comment:
Use assertArrayEquals here to compare array equality and avoid the deprecation compile warning.